Documentation/api.md

```
@@ -369,6 +371,21 @@ The `NodeExporterCollectorNetDevConfig` resource works as an on/off switch for t
 
 [Back to TOC](#table-of-contents)
 
+## NodeExporterCollectorNvmeSubsystemConfig
+
+#### Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)